extensions/ql-vscode/src/common/commands.ts

```diff
@@ -129,6 +130,7 @@ export type LocalQueryCommands = {
   "codeQL.runQueryOnMultipleDatabasesContextEditor": (
     uri?: Uri,
   ) => Promise<void>;
+  "codeQLQueries.runLocalQueryFromQueriesPanel": (uri?: Uri) => Promise<void>;
```

Comment:
Is there a reason for the separation of these two commands? They seem pretty related to me and I would expect both `codeQLQueries` commands to be in the same type.

Reply:
The command here is a new replica of the `runQuery` command that is defined in the local-queries and therefore defined here in `LocalQueryCommands`. It is executed in the queries-panel when a user 'executes' `"codeQLQueries.runLocalQueryContextInline"` through the panel. That's why we have 2 different commands. 

I suppose we could adjust the original implementation in `runQuery` to handle the case when no uri is supplied but a QueryPanelTreeViewItem instead and leave the queries-panel out of it?..